.github/workflows/notify_ms_teams.yml

```diff
@@ -0,0 +1,72 @@
+name: Notify Teams on Push
+on:
+  push:
+    branches:
+      - main
+
+env:
+  environment: dev
+
+jobs:
+  notify-teams:
+    runs-on: ubuntu-latest
+    environment:
+      name: dev
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Extract Commit Information
+        id: commit_info
+        run: |
+          COMMIT_MESSAGE=$(git log --format=%B -n 1 ${{ github.sha }})
+          COMMIT_SHA=$(git log --format=%H --abbrev=7 -n 1 ${{ github.sha }})
+          COMMIT_TIMESTAMP=$(git log --format=%ci -n 1 ${{ github.sha }})
+          COMMIT_AUTHOR=${{ github.actor }}
+
+          echo "Commit Message: $COMMIT_MESSAGE"
```

Comment:
Do we need to echo the response,  provided they are being stored in env variable?

Reply:
I can remove this I just thought this would be helpful logging to have within this action

Reply:
Yes please, if we can remove it to keep the file concise.